lib/filewatch/tail.rb

```diff
@@ -240,8 +240,24 @@ def _sincedb_write
     def quit
       _sincedb_write
       @watch.quit
+      close_files
     end # def quit
 
+    public
+    def close_file(path)
+      @watch.unwatch(path)
+      file = @files.delete(path)
+      return if file.nil?
+      _sincedb_write
```

Comment:
Needs comment doc. `close_file` is public and it is meant to be used when the external code knows that it is done with the file but has not quit or otherwise stopped tailing other files.